.github/workflows/build.yml

```diff
@@ -0,0 +1,78 @@
+name: Build
+
+on:
+  push:
+    paths-ignore:
+    - 'docs/**'
+  pull_request:
+    paths-ignore:
+    - 'docs/**'
+
+jobs:
+
+  formatter:
+    runs-on: ubuntu-latest
+    name: Check java format
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v1
+      with:
+        fetch-depth: 1
+    - uses: m273d15/java-format-action@v1.1-rc
```

Comment:
The old formatting action also contained a linter for javascript (ESLint). This is no longer the case with this action.

I am not sure what exactly the linter did an whether it can adequately be replaced through codacy rules (which also has ESLint integration).

Even though this is currently not necessary, with the re-introduction of javascript (or rather typescript) components through the VSCode implementation, this will become useful again.

Reply:
We have to think about how we want to integrate eslint, but there is a github action for executing eslint and codacy also provides a mechanism to execute eslint with a config in your repository.